starlette/testclient.py

```diff
@@ -268,8 +268,8 @@ def __init__(self, app: ASGI3App, scope: Scope) -> None:
         self.app = app
         self.scope = scope
         self.accepted_subprotocol = None
-        self._receive_queue = queue.Queue()  # type: queue.Queue
-        self._send_queue = queue.Queue()  # type: queue.Queue
+        self._receive_queue: queue.Queue = queue.Queue()
```

Comment:
Are those needed?

Reply:
Yes, otherwise:
```
+ mypy starlette tests
starlette/testclient.py:271: error: Need type annotation for '_receive_queue'
starlette/testclient.py:272: error: Need type annotation for '_send_queue'
```

Reply:
I think these (and `asyncio.Queue`) are (or at least were) needed because the elements are typed, but only in the mypy typestubs on Python 3.9+.

So on Python 3.9 you can do:
```python
foo: queue.Queue[str] = queue.Queue()
```

but on earlier versions you had to do
```python
foo: "queue.Queue[str]" = queue.Queue()
```
mypy wants the element type but can't infer it (or that it's `Any`) unless you explicitly set the type.

Reply:
We could maybe do this to make it clearer:
```suggestion
        self._receive_queue: "queue.Queue[typing.Any]" = queue.Queue()
```

Reply:
Not here, but in the future, should we consider using [those](https://github.com/django/asgiref/blob/main/asgiref/typing.py)? 

Like: `self._receive_queue: "queue.Queue[ASGIReceiveEvent]" = queue.Queue()`

Reply:
OK, I applied this change and did the same for `asyncio.Queue` further down.